### ISSUE TITLE
New Achievements and Badges: Earned X Pieces of Equipment

### ISSUE THREAD

**Issue by beffymaroo** [HabitRPG/habitica#8618]
These new achievements and badges are ready to be implemented! These should be received when a player earns the number of pieces of equipment indicated by each badge. The badge images and the names/mouseover text for the user achievement page can be found here: 

https://trello.com/c/F7a1Ic6j/8-earned-x-pieces-of-equipment-badges

**Comment by savoie** [comment]
I'd like to give this a shot :) I have so much equipment and now I've seen this issue I want the shiny badges that could come with it.

**Comment by Alys** [comment]
@savoie Thanks! Go for it! :)

**Comment by savoie** [comment]
I have _no idea_ why I thought it would be a good idea to say I could do this 2 days before starting a course-overloaded school term. I do have a few free days now though, so I should have this done today or tomorrow.

**Comment by lemoness** [comment]
@savoie - Hi there! Just checking in -- what's the status of this?

**Comment by lemoness** [comment]
This is now open for someone to work on again!

**Comment by TNychka** [comment]
Hey! I can take a look at this!

**Comment by saraolson** [comment]
Staff are looking at implementing new user achievements, so we're putting this issue on hold for now.

**Comment by veeeeeee** [comment]
We are now tracking these somewhere else to ensure proper handling.
